.travis.yml

```diff
@@ -93,7 +93,17 @@ matrix:
           packages:
           - binutils-dev
           - g++-6
-    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_PIRANHA="yes" WITH_COVERAGE="yes" TEST_IN_TREE="yes" WITH_FLINT="yes" WITH_FLINT_DEV="yes" WITH_MPC="yes" BUILD_BENCHMARKS="no" MAKEFLAGS="-j2"
+    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_COVERAGE="yes" TEST_IN_TREE="yes" WITH_FLINT="yes" WITH_FLINT_DEV="yes" WITH_MPC="yes" BUILD_BENCHMARKS="no" MAKEFLAGS="-j2"
+      compiler: gcc
+      os: linux
+      addons:
+        apt:
+          sources:
+          - ubuntu-toolchain-r-test
+          packages:
+          - binutils-dev
+          - g++-4.7
+    - env: BUILD_TYPE="Debug" WITH_BFD="yes" WITH_PIRANHA="yes" WITH_COVERAGE="yes" MAKEFLAGS="-j2"
```

Comment:
Yes, also don't add a new test. add `WITH_PIRANHA=yes` to an existing test like https://github.com/symengine/symengine/blob/master/.travis.yml#L86